src/GlobalLoop.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace React\EventLoop;
+
+final class GlobalLoop
+{
+    /**
+     * @internal
+     *
+     * @var LoopInterface
+     */
+    public static $loop;
```

Comment:
This is currently public for 2 reasons:

1. It allows to save a function call in the function API, see eg. https://github.com/reactphp/event-loop/pull/77/files#diff-84b928a9905378810b602c8a740b4cbaR16
2. It allows to swap out the loop, see eg. https://github.com/reactphp/event-loop/pull/77/files#diff-6872a075db159cbbcfc073b693c7d460R18

Note, that the property is marked `@internal`, so it should be pretty clear the messing around with it isn't a good idea ;)

Again, this is just a POC, so we might decide to make it private, always use `GlobalLoop::get()` and add a corresponding `GlobalLoop::set()` for swapping out the loop, eg. in tests.